.travis.yml

```diff
@@ -1,40 +1,31 @@
 sudo: required
 language: go
 go:
-    - "1.12"
-
+- '1.12'
 services:
-  - docker
-
+- docker
 install: true
-
-script: 
-  - VERBOSE=true make test int 
-  - ./travis-ci.sh 
-
+script:
+- VERBOSE=true make test int
```

Comment:
Most of these changes are due to the fact that the `travis` cli tool edits/rewrites the `.travis.yml` file. So it is probably just a serialization improvement even if both are technically valid.